projects/1.16/assets/prehistoric-delight-by-nocube/pre_delight/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+  "item.pre_delight.amber_sashimi": "琥珀盖饭",
+  "item.pre_delight.ammonite_escargot": "奶油烤菊石",
+  "item.pre_delight.ancient_sea_chowder": "古海浓汤",
+  "item.pre_delight.ankylo_dish": "甲龙套餐",
+  "item.pre_delight.baked_tuber": "烤块茎",
+  "item.pre_delight.boiled_tubers": "煮块茎",
+  "item.pre_delight.ceratopsian_horn_knife": "角龙犄角刀",
+  "item.pre_delight.cladophlebis_bale": "枝脉蕨捆",
```

```suggestion
  "item.pre_delight.cladophlebis_bale": "枝脉蕨块",
```
1.19.3以下原版为`干草块`，与原版术语保持一致